lib/admissions/registrar.ex

```diff
@@ -14,12 +14,20 @@ defmodule Admissions.Registrar do
   """
 
   @spec eligible?(String.t(), String.t()) :: boolean()
-  def eligible?(nickname, token), do: nickname in contributors(token)
+  def eligible?(nickname, token) do
```

Comment:
I used `any?/2` for all of these so we'd short circuit the minute we had a hit, hopefully reducing the number of API requests we have to make.